lib/foods.js

```diff
@@ -59,3 +75,33 @@ function raiseErrors(name, calories) {
     $('#calories-field .validation-error').html('Please Enter Calories');
   }
 }
+
+function setFoods() {
+  if (localStorage.getItem('foodList')) {
+    return JSON.parse(localStorage.getItem('foodList'));
```

Comment:
you're finding `localStorage.getItem('foodList')` on the line above so rather than writing it twice, i suggest setting it as a variable above 👍